src/scripts/main.js

```diff
@@ -1,3 +1,124 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const table = document.querySelector('table tbody');
+  const formContainer = document.createElement('div');
+
+  formContainer.className = 'form-container';
+
+  formContainer.innerHTML = `
+    <form class="new-employee-form">
+      <label>Name: <input name="name" type="text" data-qa="name"></label>
+      <label>Position: <input name="position" type="text" data-qa="position"></label>
+      <label>Office:
+        <select name="office" data-qa="office">
+          <option>Tokyo</option>
+          <option>Singapore</option>
+          <option>London</option>
+          <option>New York</option>
+          <option>Edinburgh</option>
+          <option>San Francisco</option>
+        </select>
+      </label>
+      <label>Age: <input name="age" type="number" data-qa="age"></label>
+      <label>Salary: <input name="salary" type="number" data-qa="salary"></label>
+      <button type="button">Save to table</button>
+    </form>
+  `;
+
+  document.body.appendChild(formContainer);
+
+  const form = formContainer.querySelector('.new-employee-form');
+  const saveButton = form.querySelector('button');
+
+  saveButton.addEventListener('click', function () {
+    const n = form.name.value.trim();
+    const position = form.position.value.trim();
+    const office = form.office.value;
+    const age = parseInt(form.age.value, 10);
+    const salary = parseFloat(form.salary.value);
```

Comment:
Consider adding validation for the salary input to ensure it is a positive number. This would prevent any negative or invalid salary values from being added to the table.